templates/go/main.go.twig

```diff
@@ -1,6 +1,42 @@
-package {{ spec.title | caseLower }}
+package main
 
-// NewClient initializes a new {{ spec.title }} client
-func NewClient() Client {
-	return Client{}
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/{{sdk.gitUserName}}/sdk-for-go/appwrite"
+)
+
+const (
+	EmptyParentDocument     = ""
+	EmptyParentProperty     = ""
+	EmptyParentPropertyType = ""
+)
+
+func main() {
+	var EmptyArray = []interface{}{}
+
+	client := appwrite.NewClient(10 * time.Second)
+	client.SetEndpoint(os.Getenv("APPWRITE_ENDPOINT"))
```

Comment:
```suggestion
	client.SetEndpoint(os.Getenv("YOUR_ENDPOINT"))
```

Change in other places similarly (YOUR_PROJECT_ID instead of APPWRITE_PROJECT and so on.